packages/smart-wallet/src/offers.js

```diff
@@ -112,6 +112,16 @@ export const makeOfferExecutor = ({
         );
         logger.info(id, 'seated');
 
+        // NB: the following four promises all resolve exactly when the seat exits.
```

Comment:
@Chris-Hibbert is this true in general? @dckc has a counter example https://github.com/Agoric/agoric-sdk/issues/6777#issuecomment-1379417692

Reply:
The way I read @dckc's comment, it was saying that if the seat doesn't exit, then various things don't work. That's consistent with my expectation that `numWantsSatisfied()`,  `getFinalAllocation()`, and `getPayouts()` resolve at the same time that the exitSubscriber reports that it's done.  The comment above seems to include `getOfferResult()`, which @erights pointed out often resolves at a different time.